walk-in-interview/src/main/webapp/common-functions.js

```diff
@@ -81,5 +83,6 @@ function getCookie(cname) {
   return '';
 }
 
-export {JOB_ID_PARAM, getRequirementsList, setErrorMessage,
+export {JOB_ID_PARAM, DEFAULT_PAGE_INDEX, DEFAULT_PAGE_SIZE,
```

Comment:
im not sure it's wise to export constants like this. You should namespace them

Reply:
will make a separate PR to do such code clean up